stefaniaPopescu26/assignment1/KthToLast.java

```diff
@@ -0,0 +1,154 @@
+/*
+    In case of k being greater than the size of
+    the list, my code will return null.
+ */
+
+public class KthToLast {
```

Comment:
There are a few issues with your solution: 
1. if you run getKthToLast with an empty list it will throw NPE, can you debug it and find a reason why?
2. for k equal exactly the size of the list your algorithm returns value 0 instead of null, can you see why this happens?

To give you a hint, this seems to be connected to the fact that you decided store a Node head in the MyLinkedList. Can you explain what is your reasoning behind having it?

If anything is not clear feel free to ask me for more clarifications :)

Reply:
I wanted head to be a sentinel, but I got lost a little :D I reviewed my code and I think this time it's better.